wasmtime/_func.py

```diff
@@ -141,7 +141,13 @@ class Caller(object):
     def __init__(self, ptr):
         self.__ptr__ = ptr
 
-    def get_export(self, name):
+    def __getitem__(self, name):
```

Comment:
Also it would be nice to have a docstring explaining how this method is different from `get` and which exact exception it raises. Although it does follow the convention, not all libraries do.